packages/xo-server/src/xo-mixins/xen-servers.mjs

```diff
@@ -165,6 +165,9 @@ export default class {
       // if value is null, pass undefined to the model , so it will delete this optionnal property from the Server object
       server.set('httpProxy', httpProxy === null ? undefined : httpProxy)
     }
+    if (fallbackAddresses !== undefined) {
+      server.set('fallBackAddresses', JSON.stringify(fallbackAddresses))
```

Comment:
Logic:
- if `null`, delete the property
- otherwise serialize it to JSON and deserialize it as necessary: https://github.com/pulls/review-requested